src/NUnitFramework/framework/Internal/Filters/CompositeFilter.cs

```diff
@@ -26,13 +25,13 @@ public CompositeFilter()
         /// <param name="filters"></param>
         public CompositeFilter( params TestFilter[] filters )
         {
-            Filters = new ReadOnlyCollection<TestFilter>(filters);
+            Filters = filters;
         }
 
         /// <summary>
         /// Return a list of the composing filters.
         /// </summary>
-        public IList<TestFilter> Filters { get; }
+        public TestFilter[] Filters { get; }
```

Comment:
array is fastest to traverse and supports `foreach` as the fastest alternative, `internal` type so signature change shouldn't matter